executor/executor.go

```diff
@@ -1,24 +1,28 @@
 package executor
 
 import (
+	"bufio"
 	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
 	"os/exec"
-	"syscall"
+	"regexp"
+	"strconv"
+	"strings"
 
+	"github.com/kr/pty"
```

Comment:
We should use glide if we are going to depend on an external package. That will prevent us from immediately breaking when master breaks upstream.